### DISCUSSION TITLE
Is it possible to skip `json_deserializer` for a particular query?

### DISCUSSION THREAD

**Discussion by connebs** [sqlalchemy/sqlalchemy#6010] [Usage Questions] [answered]
Basically title. I'm using [orjson](https://github.com/ijl/orjson) to deserialize JSON responses from Postgres 90% of the time:

```py
async_sqlalchemy_engine = create_async_engine(
    config.ASYNC_SQLALCHEMY_POSTGRES_URL,
    json_deserializer=orjson.loads,
)
```

Sometimes though I just want the JSON to be returned as the string that the engine is getting back in the first place, without deserializing.

My solution right now is to just *not* deserialize JSON ever, by doing this

```py 
async_sqlalchemy_engine = create_async_engine(
    config.ASYNC_SQLALCHEMY_POSTGRES_URL,
    json_deserializer=lambda json_str: json_str,
)
```

And then just manually deserializing when I need to (which again is 90% of the time).

But I would really prefer it if I could just not deserialize JSON for a particular `session.scalar()`.

**Comment by zzzeek** [comment, marked as answer]
I think you can use cast() or type_coerce() to String on a per-column basis and that should disable it from being treated as JSON.    depends on what kind of query you are doing.

**Reply by connebs** [reply]
Ah yeah, that makes sense. Thanks!
